### PR TITLE
Coupon Component

### PR DESCRIPTION
**Pull request by roggc**
Create a Coupon component on Storybook

### CODE REVIEW

**Comment by roggc** [comment]
coupon

**Comment by ralph-dev** [comment]
Text1 etc are not good variable names. You want to be generic but not too abstract where no one knows what's going on.

Replace text1 with CouponTitle or Title
Text2 with coupon description or Description etc

File: src/Containers/Coupon.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    Responsive,
+    MainInterface,
+    ResponsiveInterface,
+} from '../Utils/BaseStyles';
+
+export interface CouponProps
+    extends MainInterface,
+        ResponsiveInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    bg: sring;
```
Comment (ralph-dev) [inline src/Containers/Coupon.tsx:15]: color instead would be better

File: src/Containers/Coupon.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    text1: string;
+    text2: string;
+    text3: string;
+}
+
+export const Coupon: React.FC<CouponProps> = ({
+    text1,
+    text2,
+    text3,
+    ...rest
```
Comment (ralph-dev) [inline src/Containers/Coupon.tsx:25]: ...props is the standard

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Prop naming needs more work

**Comment by roggc** [comment]
coupon2

**Review by ralph-dev — APPROVED** [review]
Lgtm

**Pull request merged**